pkg/controller/plan/kubevirt.go

```diff
@@ -1721,6 +1717,13 @@ func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume,
 				Value: vm.RootDisk,
 			})
 	}
+
+	environment = append(environment,
+		core.EnvVar{
+			Name:  "V2V_NewName",
+			Value: vm.NewName,
```

Comment:
shouldn't we set it only when a new name is set? you check if V2V_NewName is set in the entrypoint of the conversion pod